DEVELOPING.md

```diff
@@ -53,3 +53,41 @@ When designing a forest for a new statistical task, we suggest following the for
 To avoid the overhead of setting up new classes and Rcpp bindings, we provide a template for a 'custom' forest. To get started, fill in the implementation for CustomRelabelingStrategy and CustomPredictionStrategy. By default, the custom forest uses the standard regression splitting rule -- if you'd like to change this, consult the ForestTrainers.custom_trainer method.
 
 This forest is made available in R as `custom.forest` (with `predict.custom.forest`). You can find a starter template for a test exercising the forest in `testthat/test_custom_forest.R`. Note that you'll need to re-run `build_package.R` after making changes to the C++ source.
+
+### Details
```

Comment:
This is really helpful! For simplicity, I think we could just call this section 'Tree Splitting Algorithm' instead of 'Details' and remove the 'SplittingRule' header below.